Release v0.0.1

nbey opened the pull request:
- create `MailgunMailer` class and config files

nbey committed:
create MailgunMailer class and config files

themightychris reviewed (changes requested):
@nbey great work, there are only two changes I'd like to see before merging:

- Rename class to `Emergence\Mailgun\Mailer`, reflecting the namespace implied by the repository name and giving us space to expand with more mailgun integration
- Break out `Emergence\Mailgun\API` as a wrapper for the API call method and try to follow the conventions laid out in [`Emergence\Slack\API`](https://github.com/JarvusInnovations/emergence-slack/tree/master/php-classes/Emergence/Slack) and more specifically [`Emergence\GitHub\API`](https://github.com/JarvusInnovations/emergence-github/tree/master/php-classes/Emergence/GitHub) -- I like how you broke out the two config fragments, but lately with API wrappers I've leaned towards putting the core required config params together in the "main" config file since they have to be set and it doesn't make sense to set them separately 
- use short array syntax throughout (e.g. `array $foo = []`)

nbey committed:
Use short array syntax

nbey committed:
refactor: break out API and Mailer classes

- use "main" config file for needed configs
- rename `Emergence\Mailer\Mailgun` -> `Emergence\Mailgun\Mailer`

nbey committed:
fix: remove deprecated API configurations

nbey commented:
@themightychris Could you give this another look?

nbey committed:
fix: remove empty strings
